### PR TITLE
Add basic app structure.

### LINKED ISSUE #2: Setup App Basic Structure.

### PR DESCRIPTION
**Pull request by krystian8207**
Closes: #2

### CODE REVIEW

**Comment by krystian8207** [comment]
Two new warnings appeeard.
Seems to be not fixable:
1. https://stackoverflow.com/questions/69430064/this-font-awesome-icon-gears-does-not-exist-in-r-shiny
2. https://stackoverflow.com/questions/39196743/interactive-directory-input-in-shiny-app-r

File: R/devops-package.R [hunk 1]
```diff
@@ -0,0 +1,4 @@
+#' devops package
+#' @name devops-package
+#' @importFrom magrittr %>%
```
Comment (igrasek) [inline R/devops-package.R:3]: @krystian8207  What is this for?

**Review by igrasek — CHANGES_REQUESTED** [review]
Where readme?

File: R/devops-package.R (hunk 1 above)
Comment (krystian8207) [reply R/devops-package.R:3]: This is to make pipe (`%>%`) work.

**Review by igrasek — APPROVED** [review]
lgtm

**Review by igrasek — APPROVED** [review]
further changes accetep as well

**Pull request merged**